queries.sql

```diff
@@ -48,3 +48,10 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, SUM(escape_attempts) as total_escape_attempts FROM animals GROUP BY neutered ORDER BY total_escape_attempts DESC;
 SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) AS avg_escape_attempts FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+SELECT a.name AS Animals_of_Melody_Pong FROM animals AS a JOIN owners AS o ON a.owner_id = o.id WHERE o.full_name = 'Melody Pond';
+SELECT a.name AS Animal_of_type_Pokemon FROM animals AS a JOIN species AS s ON a.species_id = s.id WHERE s.name = 'Pokemon';
```

Comment:
- [ ] The query is returning 6 animals, including "Devimon," that are classified as "pokemon." However, [it appears that](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Modify%20your%20inserted,will%20be%20Digimon) "Devimon" should belong to the "digimon" species as its name ends with "mon." Please update your code and ensure that the query returns the correct values in line with the project requirements.
![image](https://github.com/enocol/vet_clinic_database/assets/55840999/75ac9c17-e76e-47eb-a7b7-ac3ad5d6c7ab)

- [ ] As you haven't included the insert queries in this milestone, it's currently impossible to verify not only this query but also others. I recommend adding the insert queries and then rerunning the select queries to verify their accuracy. Once you've done this, update the screenshots if necessary. 📊📝🐾

Reply:
@shahadat3669  Hello, please I will need you to exolain abit further what you mean here about INSERT QUERIES. Thanks